### ISSUE TITLE
Remove dependency on doctrine/common

### ISSUE THREAD

**Issue by deresh** [doctrine/data-fixtures#300]
Newer versions od doctrine deprecated doctrine/common package. 

Please remove this dependency and depend on new doctrine packages as needed.

**Comment by shehi** [comment]
That'd be breaking for a lot of bundles that use the current version of this package, bundles that rely on doctrine-common. E.g. Faker uses doctrine-common to abstract the usages of ORM and ODM loaders.

**Comment by alcaeus** [comment]
@shehi that's wrong: all non-deprecated functionality has been extracted into other packages. If any library relies on transitive dependencies (e.g. import doctrine/data-fixtures and expect it to import doctrine/common), this is bad design on their part.

We will be phasing out the dependency to doctrine/common in the near future.

**Comment by shehi** [comment]
You will be transitioning them out in doctrine/data-fixtures 1.x or next major release? I am asking just to be 100% sure what you mean.

**Comment by alcaeus** [comment]
This interface has been moved to `doctrine/persistence` a while back, with `doctrine/common` requiring `doctrine/persistence`. This issue is about removing `doctrine/common` and replacing it with the individual packages. This will be done in a 1.x release as it doesn't constitute a breaking change as long as we don't change our public API.
